plotly/tools.py

```diff
@@ -352,35 +352,36 @@ def embed(file_owner_or_url, file_id=None, width="100%", height=525):
     file_id (default=None) -- an int or string that can be converted to int
                               if you're using a url, don't fill this in!
     width (default="100%") -- an int or string corresp. to width of the figure
-    height (default="525") -- same as width but corresp. to the height of the figure
+    height (default="525") -- same as width but corresp. to the height of the
+                              figure
 
     """
     try:
-        s = get_embed(file_owner_or_url, file_id, width, height)
+        s = get_embed(file_owner_or_url, file_id=file_id, width=width,
+                      height=height)
+
         # see if we are in the SageMath Cloud
         from sage_salvus import html
         return html(s, hide=False)
     except:
         pass
     if _ipython_imported:
-        if file_id:
-            plotly_domain = (
-                session.get_session_config().get('plotly_domain') or
-                get_config_file()['plotly_domain']
-            )
-            url = "{plotly_domain}/~{un}/{fid}".format(
-                plotly_domain=plotly_domain,
-                un=file_owner_or_url,
-                fid=file_id)
-        else:
-            url = file_owner_or_url
+        url = file_owner_or_url
```

Comment:
:cactus: we can't just set `file_owner_or_url` to url! We check `file_id` because:

* `if file_id is not None` we assume `file_owner = file_owner_or_url`
* `if file_id is None` we assume `url = file_owner_or_url`

`get_embed` is public and available to users, any code that might be out there using a username instead of a url would break because of this, right?